src/bundle/Resources/public/js/scripts/core/collapse.js

```diff
@@ -1,21 +1,97 @@
-(function (global, doc) {
+(function (global, doc, bootstrap, Translator) {
+    let toggleAllTimeout;
+    const toggleAllBtns = [...doc.querySelectorAll(`[data-multi-collapse-btn-id]`)];
+    const toggleMultiCollapseBtn = (btn, changeToCollapseAll) => {
+        const displayedText = changeToCollapseAll
+            ? /*@Desc("Collapse all)*/ 'product_type.edit.section.attribute_collapse_all'
+            : /*@Desc("Expand all)*/ 'product_type.edit.section.attribute_expand_all';
+
+        btn.innerText = Translator.trans(displayedText, {}, 'ibexa_product_catalog');
+        btn.classList.toggle('ibexa-multi-collapse__btn--expand-all-label', !changeToCollapseAll);
+    };
+
     doc.querySelectorAll('.ibexa-collapse').forEach((collapseNode) => {
         const toggleButton = collapseNode.querySelector('.ibexa-collapse__toggle-btn');
         const isCollapsed = toggleButton.classList.contains('collapsed');
 
         collapseNode.classList.toggle('ibexa-collapse--collapsed', isCollapsed);
         collapseNode.dataset.collapsed = isCollapsed;
 
+        const multiCollapseNode = collapseNode.closest(`[data-multi-collapse-body]`);
+
         collapseNode.addEventListener('hide.bs.collapse', (event) => {
             event.stopPropagation();
+
             collapseNode.classList.add('ibexa-collapse--collapsed');
             collapseNode.dataset.collapsed = true;
         });
 
         collapseNode.addEventListener('show.bs.collapse', (event) => {
             event.stopPropagation();
+
             collapseNode.classList.remove('ibexa-collapse--collapsed');
             collapseNode.dataset.collapsed = false;
         });
+
+        if (!multiCollapseNode || !toggleAllBtns.length) {
+            return;
+        }
+
+        const currentToggleAllBtn = doc.querySelector(`[data-multi-collapse-btn-id="${multiCollapseNode.dataset.multiCollapseBody}"]`);
+        const attachClickToggleHandler = (section) => {
+            section.addEventListener('click', () => {
+                const currentCollapsibleBtns = [...multiCollapseNode.querySelectorAll('[data-bs-toggle]')];
+
+                window.clearTimeout(toggleAllTimeout);
```

Comment:
here and other places: You have `window` under `global` 
```suggestion
                global.clearTimeout(toggleAllTimeout);
```